test/acceptance/view_benchmark_graphs_test.rb

```diff
@@ -72,9 +72,60 @@ class ViewBenchmarkGraphsTest < AcceptanceTest
     assert_not page.has_content?("#{benchmark_run_category_humanize} memory Graph")
   end
 
+  test "User should be able to view long running benchmark graphs when they go back and forward 
+    in the broswer window" do
+
+    javascript_driver = Capybara.javascript_driver
+    default_driver = Capybara.current_driver
+    Capybara.javascript_driver = :selenium
+    Capybara.current_driver = :selenium
```

Comment:
Ah ok I see what you mean. I think the problem is because we're choosing two radio buttons one after another. How about doing something like this?

```
within "form" do
  choose(benchmark_run.benchmark_type.category)
end

assert page.has_content?("#{benchmark_run_category_humanize} Graph")
...

within "form" do
  choose(benchmark_run2.benchmark_type.category)
end

assert page.has_content?("#{benchmark_run2_category_humanize} Graph")
...

page.evaluate_script(window.back())
```

This way we'll be able to not depend on selenium which is alot slower.